PencilCase/PencilCase/www/css/custom.css

```diff
@@ -0,0 +1,3 @@
+﻿.ui-mobile, .ui-mobile body {
```

Comment:
Override such settings in jquery mobile css, which is 99.9%, here just set it as unset.